File: languages/tree-sitter-stack-graphs-java/test/statement/try_statement.java
class A {
  void f() {
    try {
        f();
      //^ defined: 2
    } catch (A e) {
        e;
      //^ defined: 6
        f();
      //^ defined: 2
    } finally {
        f();
      //^ defined: 2
    }
  }
}

File: languages/tree-sitter-stack-graphs-javascript/test/statements/try_statement.js
let x = 1;

// Flow in

try {
    /**/ x;
    //   ^ defined: 1
    y = 1;
} catch (e) {
    /**/ e
    //   ^ defined: 9
    /**/ y;
    //   ^ defined: 8
    y = 1;
} finally {
    /**/ x;
    //   ^ defined: 1
    /**/ y;
    //   ^ defined: 8, 14
    y = 1;
}

// Flow out

/**/ y;
//   ^ defined: 8, 14, 20

// Flow around

/**/ x;
//   ^ defined: 1